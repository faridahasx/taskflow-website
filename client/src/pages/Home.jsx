import { TaskProvider } from "../context/TaskContext";
import Layout from "../components/Layout/Layout";
import TasksContainer from "../containers/Tasks/TasksContainer";

const Home = () => {
  return (
    <TaskProvider>
      <Layout>
        <TasksContainer />
      </Layout>
    </TaskProvider>
  );
};

export default Home;
