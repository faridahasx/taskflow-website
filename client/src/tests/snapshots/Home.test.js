import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";
import Home from "../../pages/Home";
import mockInitialState, { mockStore } from "../mocks/mockReduxState";


test("Home snapshot", () => {
  const { asFragment } = render(
    <Provider store={mockStore(mockInitialState)}>
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    </Provider>
  );
  expect(asFragment()).toMatchSnapshot("Home");
});
