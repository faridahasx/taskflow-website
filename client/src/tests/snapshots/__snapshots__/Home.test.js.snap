// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot: Home 1`] = `<DocumentFragment />`;
